cephfs/admin/subvolume_test.go

```diff
@@ -688,3 +688,33 @@ func TestSubVolumeSnapshotInfo(t *testing.T) {
 	assert.Error(t, err)
 	assert.Nil(t, sinfo)
 }
+
+func TestSubvolumeQuiesce(t *testing.T) {
+	fsa := getFSAdmin(t)
+	volume := "cephfs"
+	group := NoGroup
+	fsa.CreateSubVolume(volume, group, "quiesceMe", nil)
+	defer func() {
+		err := fsa.RemoveSubVolume(volume, group, "quiesceMe")
+		assert.NoError(t, err)
+	}()
+	ret, err := fsa.SubVolumeQuiesce(volume, group, []string{"quiesceMe"}, "", nil)
```

Comment:
because this is a new Ceph Mgr API call that is not available in current Ceph versions, you will need to move the `TestSubvolumeQuiesce()` function into it's own `subvolume_squid_test.go` file and add a Go build-tag like

```go
//go:build ceph_squid
```

on the first line of that file.

In  `subvolume_reef_test.go` you can have this function, but you should expect a failure when `SubVolumeQuiesce()` is executed. You should then verify that the error is of type `ErrNotImplemented` or whetever you think it should be. 

The `subvolume_reef_test.go` should have this in the 1st line:

```go
//go:build !ceph_squid
```

Reply:
I have created the test files, I have named tests SubVolumeQuiesceReef and SubVolumeQuiesceSquid